pkgs/racket-doc/scribblings/reference/promise.scrbl

```diff
@@ -173,3 +173,9 @@ TODO: Say something on:
   demand.
 ;}
 }
+
+@defform[(delay* body ...+)]{
+Like @racket[delay], except that it only handles one value, and it
+doesn't deal with exceptions.
```

Comment:
This should specify what happens if the `body` expressions produce more than one value, or if they raise an exception.